plugins/bgnotify/bgnotify.plugin.zsh

```diff
@@ -85,23 +86,15 @@ function bgnotify {
       esac
     fi
 
-    if [[ -z "$term_id" ]]; then
-      terminal-notifier -message "$message" -title "$title" &>/dev/null
-    else
-      terminal-notifier -message "$message" -title "$title" -activate "$term_id" -sender "$term_id" &>/dev/null
-    fi
+    terminal-notifier -message "$message" -title "$title" ${=icon:+-appIcon "$icon"} ${=term_id:+-activate "$term_id" -sender "$term_id"} &>/dev/null
   elif (( ${+commands[growlnotify]} )); then # macOS growl
     growlnotify -m "$title" "$message"
```

Comment:
Not part of this PR, but is this software used anymore? A somewhat quick search suggests that growlnotify was abandoned some years ago.

Reply:
Let's keep it. Maybe someone is still using it. We support zsh down to 5.0.8 (released in mid-2015).